Gemfile

```diff
@@ -14,63 +14,33 @@ group :documentation do
   gem 'relish', '~> 0.7.1'
 end
 
-platforms :jruby do
-  gem "jruby-openssl"
-end
+gem 'rake', '~> 12'
 
-case RAILS_VERSION
-when /main/
-  MAJOR = 6
-  MINOR = 0
-when /5-2-stable/
-  MAJOR = 5
-  MINOR = 2
-when /stable/
-  MAJOR = 6
-  MINOR = 0
-when nil, false, ""
-  MAJOR = 6
-  MINOR = 0
-else
-  match = /(\d+)(\.|-)(\d+)/.match(RAILS_VERSION)
-  MAJOR, MINOR = match.captures.map(&:to_i).compact
+if RUBY_VERSION.to_f >= 2.3
+  gem 'rubocop', '~> 0.80.1'
 end
 
+gem 'capybara'
+
+MAJOR =
+  case RAILS_VERSION
+  when /5-2-stable/
+    5
+  when /main/, /stable/, nil, false, ''
+    6
+  else
+    /(\d+)[\.|-]\d+/.match(RAILS_VERSION).captures.first.to_i
+  end
+
 if MAJOR >= 6
-  gem 'selenium-webdriver', '~> 3.5', require: false
+  # Rails 6.0 specify '~> 1.4' directly in the adapter.
```

Comment:
Do you mean `> 1.4?` Either way we should let Rails decide what version to use?

Reply:
The comment below makes no sense. When `sqlite3` 1.5.0 is out, it still matches the [Rails' requirement](https://github.com/rails/rails/blob/6-0-stable/activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb#L13).

But if it happens that `sqlite3` decides to jump straight to 2.0, our builds will be broken if we omit this constraint.

I'll fix the comment.

Reply:
Sorry it was late when I wrote this, what I mean is if the Rails constraint is `~> 1.4` then this has no effect and can be removed. If the rails constraint is `> 1.4` then this has an effect but again I don't feel we should constrain this, we need to match Rails here.

Reply:
Point is `sqlite3` is an optional Rails dependency, they only call `gem` when the adapter is set to `sqlite` in `config/database.yml`. But still they have a version constraint defined there, which Bundler has no awareness of, since it's outside of the `Gemfile`.

If we remove it, `sqlite3` won't be installed and the build would break.
If we remove the version constraint, the build will break for Rails 6.0 when (eventually, far in the future) `sqlite3` 2.0 is out.

Reply:
Right ok, thats not clear from the comment at all.